packages/shared/registerInternalModuleStart.js

```diff
@@ -3,8 +3,6 @@
  *
  * This source code is licensed under the MIT license found in the
  * LICENSE file in the root directory of this source tree.
- *
- * @flow
```

Comment:
I guess it doesn't really matter that we'll "lose" Flow coverage on these files anyway b'c the DevTools hook doesn't have strong Flow types.